src/ComposerRequireChecker/Cli/CheckCommand.php

```diff
@@ -131,7 +133,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     private function getCheckOptions(InputInterface $input): Options
     {
         $fileName = $input->getOption('config-file');
-        if (!$fileName) {
+        if (!$fileName || !is_string($fileName)) {
```

Comment:
We should probably assert `$filenName` to be a `string`, if set (instead of silently continuing)